docassemble/StudentEvaluations/data/questions/prs_complaint_interview.yml

```diff
@@ -197,8 +197,10 @@ id: students name
 question: |
   What is the name of your child?
 fields:
-  - code: |
-      student.name_fields()
+  - First name: student.name.first
```

Comment:
See better way to fix suffix 

https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/question_library/names#optional-omit-suffix